api/controllers/v1/UserController.coffee

```diff
@@ -0,0 +1,29 @@
+# UserController
+#
+# @description :: Server-side logic for managing users
+# @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
+
+
+module.exports =
+
+  # `UserController.index()`
+
+  index: (req, res) ->
```

Comment:
OK, agreed - the convention application wide is `req` and `res`. I'll adjust the tests to follow this convention.